src/effects/Amplify.cpp

```diff
@@ -229,6 +229,13 @@ EffectAmplify::PopulateOrExchange(ShuttleGui & S, EffectSettingsAccess &)
       }
    }
 
+   // Determine the new peak, because at this stage it still has a garbage value;
+   // also, we can not use mRatio to do it (as done in other parts of the code)
+   // because at this stage mRatio is not yet reflecting the slider/textinput value
+   // (which is linked to mAmp)
+   //
+   mNewPeak = LINEAR_TO_DB(DB_TO_LINEAR(mAmp) * mPeak);
```

Comment:
1. Can you verify that commit 7f63376534dc645b1c4b78a669c6ca4266975093 introduced the bug, which did not exist before?
2. Can you fix it instead by calling ClampRatio() to avoid duplication of the logic?
3. Also make an in-class initialized of mNewPeak to 0, and for other members while you can (mRatioClip, mPeak).  Better to initialize members of built-in number types always, and better to do that with member initializers, not assignments inside the constructor.  In-class initializers are best for assisting the human reader of the code to be confident that it is right.

Reply:
1. yes, verified